apps/site/assets/ts/stop/components/DepartureList.tsx

```diff
@@ -32,22 +40,23 @@ const DepartureList = ({
   // TODO: handle no predictions or schedules case and predictions only case
   return (
     <>
-      {schedules.length && (
-        <div className="stop-departures departure-list-header">
-          <div className={`departure-card__route ${routeBgClass(route)}`}>
-            <div>
-              {renderSvg("c-svg__icon", routeToModeIcon(route), true)}{" "}
-              {routeName(route)}
-            </div>
-            <a
-              className="open-schedule"
-              href={`../schedules/${route.id}/line?schedule_direction[direction_id]=${directionId}&schedule_direction[variant]=${schedules[0].trip.route_pattern_id}`}
-            >
-              View all schedules
-            </a>
+      <div className="stop-departures departure-list-header">
+        <div className={`departure-card__route ${routeBgClass(route)}`}>
+          <div>
+            {renderSvg("c-svg__icon", routeToModeIcon(route), true)}{" "}
+            {routeName(route)}
           </div>
+          <a
+            className="open-schedule"
+            href={`../schedules/${route.id}/line?schedule_direction[direction_id]=${directionId}&schedule_direction[variant]=${schedules[0]?.trip.route_pattern_id}`}
+          >
+            View all schedules
+          </a>
         </div>
-      )}
+      </div>
+      {schedules.length === 0 &&
+        Object.keys(predictionsByHeadsign).length === 0 &&
```

Comment:
question: This will only return true if there are no predictions for any route that stops at the stop. Is it possible to only look at predictions for the selected headsign?